Header header

sensor_msgs/Image thermal
sensor_msgs/Image rgb
sensor_msgs/PointCloud2 points
